autoload/SpaceVim/layers/lang/scheme.vim

```diff
@@ -7,11 +7,22 @@
 "=============================================================================
 
 function! SpaceVim#layers#lang#scheme#config() abort
-  call SpaceVim#plugins#runner#reg_runner('scheme', 'echo | mit-scheme --quiet --load %s && echo')
+  if s:scheme_dialect ==# 'mit-scheme'
+    call SpaceVim#plugins#runner#reg_runner('scheme', 'echo | mit-scheme --quiet --load %s && echo')
+  elseif s:scheme_dialect ==# 'guile'
+    call SpaceVim#plugins#runner#reg_runner('scheme', 'echo | guile %s && echo')
+  else
+    echomsg "executing " . s:scheme_dialect . " files in the editor is currently unsupported"
```

Comment:
no need to print the message, if the command doesn't exist. runner will failed with exit code.
this function is called when vim start up. echomsg will block vim screen.

Reply:
good point, I'll fix